pxr/imaging/hd/utils.cpp

```diff
@@ -274,21 +276,37 @@ ConvertHdMaterialNetworkToHdMaterialNetworkSchema(
                 .Build());
     }
 
+    infoNames.reserve(hdNetworkMap.info.size());
+    infoValues.reserve(hdNetworkMap.info.size());
+    for (const auto& infoEntry : hdNetworkMap.info)
+    {
+      // from _dataSourceLegacyPrim.cpp - _ToContainerDS(VtDictionary)
```

Comment:
Here & elsewhere, no need to update this but if it's annoying you I think we could add VtDictionary <-> ContainerDS translators somewhere accessible (maybe retainedDataSource.h?).

Reply:
There were other places in the code where these conversions were happening, but I couldn't see a clean way to share that code.  I don't want to bite off more than I can chew here, so maybe I'll leave that for a later refacor - or if anyone wants to jump in and make that change, thats also fine.